sections/scenario_end.tex

```diff
@@ -21,8 +21,17 @@
 In \textbf{Clash} Scenarios with three or more players, collecting \textbf{a Faction Cube} from every enemy player immediately wins you the game.
 Other Scenario specific rules may also modify the outcome of collecting Faction Cubes.
 
+After finishing a \textbf{Solo Campaign} scenario, reset your Hero's Experience Level to 1, and prepare the starting deck for the next scenario of the campaign.
+It will consist of:
+\begin{itemize}
+  \item all the \hyperlink{Statistic}{Statistic} cards from your deck,
+  \item the level 1 \hyperlink{Specialty}{Specialty} card,
+  \item 5 other non-Specialty cards of your choice from your deck.
+\end{itemize}
+Skip steps 11-13 of Setup for the next scenario of the campaign.
+
 \vspace*{\fill}
 \hspace{-6em}
-\includegraphics[width=2\linewidth]{\art/medusa.png}
+\includegraphics[width=1.5\linewidth]{\art/medusa.png}
```

Comment:
Small Medusa doesn't look good. Move some of the text to the left column. If Medusa's tail will be too close to the text, move it slightly to the right by modifying `\hspace` value.

Reply:
What about now? If I don't resize it at all, it gets pushed to the next page, which is much worse.